boxsdk/object/file.py

```diff
@@ -67,6 +69,52 @@ def create_upload_session(self, file_size, file_name=None):
             response_object=response,
         )
 
+    def chunked_upload_with_stream(self, content_stream, file_size, file_name=None):
```

Comment:
This should have the `@api_call` decorator, since it makes an API call.  It might also be good to give this a name with a verb, e.g. `get_chunked_uploader_for_stream()` or something